src/gui/attribute_table.py

```diff
@@ -15,9 +21,10 @@ def toolbar(table):
         return table.findChild(QToolBar, "mToolbar")
 
     @staticmethod
-    def refresh(table, layerName):
-        table.findChild(QAction, "mActionReload").trigger()
+    def refresh(table, layerName, filterExpr="True"):
```

Comment:
can we add the typing while being on this method?

Reply:
done !